packages/core/src/components/popover/popover.tsx

```diff
@@ -300,7 +295,7 @@ export class Popover<
         }
     }
 
-    protected validateProps(props: PopoverProps<T> & { children?: React.ReactNode }) {
+    protected validateProps(props: PopoverProps<T>) {
```

Comment:
this seems fine - `{ children?: React.ReactNode }` is the same as what's on `PopoverSharedProps`